backend/lib/edgehog/triggers/handler/manual_actions/handle_trigger.ex

```
@@ -154,9 +154,16 @@ defmodule Edgehog.Triggers.Handler.ManualActions.HandleTrigger do
         network_deployment =
           Containers.fetch_network_deployment!(network_id, device.id, tenant: tenant)
 
-        if event.value,
-          do: Containers.mark_network_deployment_as_available(network_deployment, tenant: tenant),
-          else: Containers.mark_network_deployment_as_unavailable(network_deployment, tenant: tenant)
+        case event.value do
+          true ->
+            Containers.mark_network_deployment_as_available(network_deployment, tenant: tenant)
+
+          false ->
+            Containers.mark_network_deployment_as_unavailable(network_deployment, tenant: tenant)
+
+          nil ->
+            Containers.destroy_network_deployment!(network_deployment, tenant: tenant)
+        end
```

Great, now thanks to `Containers.destroy_network_deployment!` we should see when the backend fails to purge the network deployment.

However, in the first two cases, errors are silently ignored.
It would be better to use the `!` versions of the Ash-generated functions.

The same is true for the changes on the similar events for the other resources (volumes and device mappings).